.github/workflows/CHERRY_PICK.md

```diff
@@ -0,0 +1,49 @@
+# Automated Cherry-Pick Workflow
+
```

Comment:
Would be good to add some details about the motivation and purpose of this.

Something like:

> The `auto-cherry-pick.yml` workflow enables [IssueOps](https://issue-ops.github.io/docs/) for certain PR's that, when merged, should be be automatically cherry-picked into a target branch such as a release branch.